code_submitter/auth.py

```diff
@@ -1,10 +1,12 @@
-import base64
-import logging
-import binascii
-from typing import cast, List, Tuple, Optional, Sequence
 from typing_extensions import TypedDict
 
 import httpx
+import base64
+import logging
+import secrets
+import binascii
+from typing import cast, Dict, List, Tuple, Optional, Sequence
+from ruamel.yaml import YAML
```

Comment:
This is definitely wrong -- `httpx` is an external package and so should be below these standard library imports. I'll have a look at what's up with the `isort` config. I suspect this could be an version 5 related issue.

Reply:
Yeah, this was a version thing. I've sorted this in df46321a6bc2807c2e92465e7f0c694710bcefba.